### DISCUSSION TITLE
npm start [Jan 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5038] [Discord Threads]
> 🙋‍♂️ **Kuckuckmann**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Do `npm install` in the directory  
#  
> 🙋‍♂️ **Kuckuckmann**  
#### I did  
#  
> 🙋‍♂️ **Kuckuckmann**  
#### just before that message came with npm start  
#  
> 🙋‍♂️ **Kuckuckmann**  
#### you mean the nocode-seed directory, right?  
#  
> 👨‍💻 **pranavxc**  
#### yes  
#  
> 👨‍💻 **pranavxc**  
#### any error while doing `npm install`  
#  
> 🙋‍♂️ **Kuckuckmann**  
#### no it run through without any errors  
#  
> 👨‍💻 **pranavxc**  
#### Can you check the `nocodb` folder within `node_modules`  
#  
> 🙋‍♂️ **Kuckuckmann**  
#### "No such file or directory"  
#  
> 🙋‍♂️ **Kuckuckmann**  
#### it's nocodb-seed/node_modules/  
#  
> 🙋‍♂️ **Kuckuckmann**  
#### <@682854952657616902>  please don't forget me  
#  
> 👨‍💻 **pranavxc**  
#### do `npm install nocodb@latest`  
#  
> 👨‍💻 **pranavxc**  
#### can u share the `package.json` in `nocodb-seed`  
#  
> 🙋‍♂️ **Kuckuckmann**  
#### that helped, it#s installed  
#  
> 🙋‍♂️ **Kuckuckmann**  
#### how can I install it as a service?  
#  
> 🙋‍♂️ **Kuckuckmann**  
#### that it runs at systemctl automatically?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1055805509745266738)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
